docs/markdown/Machine-files.md

```diff
@@ -84,14 +84,63 @@ command line will override any options in the native file. For example, passing
 
 In addition to special data that may be specified in cross files, this
 section may contain random key value pairs accessed using the
-`meson.get_external_property()`
+`meson.get_external_property`, or `meson.get_cross_property`.
 
-## Properties
+### Project specific options
 
-*New for native files in 0.54.0*
+*New in 0.55.0*
 
-The properties section can contain any variable you like, and is accessed via
-`meson.get_external_property`, or `meson.get_cross_property`.
+Path options are not allowed, those must be set in the `[paths]` section.
```

Comment:
Or rather we could deprecate the `[paths]` section and have them into the built-in options section? I personally don't like imposing to the user arbitrary split of options. Sounds like imposing an implementation detail into our API.

Reply:
This line needs to be edited, since it contradicts the line above:
```
### Paths and Directories
*Deprecated in 0.55.0* use the built-in section instead.
```

Reply:
Now that you did deprecate [paths] that line should be removed.